src/jobs/JobsManager.ts

```diff
@@ -165,6 +165,9 @@ export class JobsManager {
     G extends keyof IQueuedJobs,
     J extends keyof IQueuedJobs[G]
   >(jobGroup: G, jobName: J, state: boolean) {
+    if (!localStorage) {
```

Comment:
Can we use a fully qualified name like `window?.localStorage`? It will make it easier to find those in code.

Reply:
Sure

Reply:
Changed